packages/theme/src/cli/commands/theme/info.ts

```diff
@@ -1,17 +1,83 @@
+import {themeFlags} from '../../flags.js'
 import ThemeCommand from '../../utilities/theme-command.js'
+import {ensureThemeStore} from '../../utilities/theme-store.js'
+import {findOrSelectTheme} from '../../utilities/theme-selector.js'
 import {themeInfo} from '../../services/info.js'
+import {DevelopmentThemeManager} from '../../utilities/development-theme-manager.js'
+import {Flags} from '@oclif/core'
 import {globalFlags} from '@shopify/cli-kit/node/cli'
-import {outputInfo} from '@shopify/cli-kit/node/output'
+import {formatSection, outputInfo} from '@shopify/cli-kit/node/output'
+import {ensureAuthenticatedThemes} from '@shopify/cli-kit/node/session'
 
 export default class ThemeInfo extends ThemeCommand {
-  static description = 'Displays information about your theme environment, including your current store.'
+  static description =
+    'Displays information about your theme environment, including your current store. Can also retrieve information about a specific theme.'
 
   static flags = {
     ...globalFlags,
+    password: themeFlags.password,
+    store: themeFlags.store,
```

Comment:
What do you think about include the [`--environment`](https://github.com/Shopify/cli/blob/98902f51404bf5ff24edeccb77a0c9220a72da4e/packages/theme/src/cli/flags.ts#L29) flag here? Then, users may rely on [`shopify.theme.toml`](https://shopify.dev/docs/themes/tools/cli/environments) to set their flags.